src/compiler/scala/tools/nsc/typechecker/Checkable.scala

```diff
@@ -39,7 +37,7 @@ import scala.tools.nsc.Reporting.WarningCategory
  *
  *  There are four possibilities to consider:
  *     [P1] X will always conform to P
- *     [P2] x will never conform to P
+ *     [P2] X will never conform to P
```

Comment:
I think this isn't a typo: I think it's from `x ∉ P              // (x ∈ $X) ⇒ (x ∉ $P)`

Reply:
I think you're right, but doesn't it need a different verb?